x/ibc/24-host/utils.go

```diff
@@ -0,0 +1,11 @@
+package host
+
+// RemovePath is an util function to remove a path from a set.
+func RemovePath(paths []string, path string) ([]string, bool) {
```

Comment:
When do we need to use this? Generally we should not be treating paths as sets.

Reply:
`removeConnectionFromClient` on ICS03

Reply:
Ah, I see. Do we have a proper set data structure we could use instead of a list?

Reply:
That might be nice...

Reply:
whats wrong with that list?